AABB_tree/include/CGAL/AABB_traits.h

```diff
@@ -289,6 +289,8 @@ class AABB_traits:
    */
   class Sort_primitives
   {
+    const Sort_primitives& operator=(const Sort_primitives&);
```

Comment:
Why the `const` return type? `operator=` usually returns a non-const reference in C++. (not that people often take advantage of it though)